agr_literature_service/api/crud/reference_crud.py

```diff
@@ -230,6 +223,28 @@ def show_all_references_external_ids(db: Session):
             for reference in references_query.all()]
 
 
+def get_merged(db: Session, curie):
+    reference = None
+    logger.debug("Looking up if '{}' is a merged entry".format(curie))
+    # Is the curie in the merged set
+    try:
+        orc: ObsoleteReferenceModel = db.query(ObsoleteReferenceModel).filter(ObsoleteReferenceModel.curie == curie).one_or_none()
+    except Exception:
+        logger.debug("No merge data found so give error message")
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND,
+                            detail=f"Reference with the id {curie} is not available")
+
+    # If found in merge then get new reference.
+    if orc:
+        logger.debug("Merge found looking up the id '{}' instead now".format(orc.new_id))
```

Comment:
I think @valearna (and Paulo) preferred f-statements like in line 235 above, and I think that's more readable, but it's probably not too important.